react/features/screenshot-capture/ScreenshotCaptureSummary.js

```diff
@@ -58,38 +61,30 @@ export default class ScreenshotCaptureEffect {
      * @returns {Promise} - Promise that resolves once effect has started or rejects if the
      * videoType parameter is not desktop.
      */
-    startEffect(stream: MediaStream, videoType: string) {
```

Comment:
Why is this no longer using the effects API?

Reply:
Because effects are used to alter the stream and then pass it further altered, and we don't need that here